acceptance/Rakefile

```diff
@@ -18,16 +18,11 @@ module HarnessOptions
     :type => 'aio',
     :helper => ['lib/helper.rb'],
     :tests  => ['tests'],
-    :log_level => 'info',
-    :color => false,
-    :root_keys => true,
     :ssh => {
       :keys => ["~/.ssh/id_rsa-acceptance"],
     },
-    :xml => true,
     :timesync => false,
```

Comment:
good call.  i'll add a comment there as to why it's false.  

I'd really prefer the rakefile to be as dumb as possible, so people can easily run from either beaker or the Rakefile.  we've had this issue on other projects.
the rakefile can provide sugar to help with some tasks like host generator, and abstracting the CI job config back into this repo.